jekyll/_cci2/migrating-from-travis.md

```diff
@@ -8,56 +8,47 @@ description: "Migrating from Travis CI"
 This document provides an overview of how to migrate from Travis CI to CircleCI.
 
 The example build configurations referenced throughout this article are based
-off this [example JavaScript
-repository](https://github.com/CircleCI-Public/circleci-demo-javascript-express/blob/master/.circleci/config.yml).
-For context (and limiting the scope of this article), consider the
-example repository's owner to want to achieve the following with their
-Continuous Integration tooling:
-
-- On pushing code: run a build and run all tests.
-- Expect that build time will be reduced by caching any dependencies after the
-  initial build.
-- Enable the safe use of environment variables.
-- On each build, upload a `test-results.xml` file to be made accessible online.
+on this [example JavaScript repository](https://github.com/CircleCI-Public/circleci-demo-javascript-express/blob/master/.circleci/config.yml).
 
 ## Prerequisites
 
-This document assumes that you have an account with CircleCI that is linked
-to a repository. If you don't, consider going over our [getting started guide]({{ site.baseurl }}/2.0/getting-started/).
+This document assumes that:
+1. You have an account with CircleCI that is linked to a repository. If you don't yet,
+consider reading our [Getting Started Guide]({{ site.baseurl }}/2.0/getting-started/).
+1. You understand the [Basic Concepts]({{ site.baseurl }}/2.0/concepts/) in CircleCI.
+
+## Why Migrate to CircleCI?
+
+- **Scaling Concurrency**: You can run up to 80 concurrent jobs on our monthly Performance Plan or even more on an [annual plan](https://circleci.com/pricing/). Travis CI has capped concurrencies of 1, 2, 5, and 10 on each of their plans.
```

Comment:
```suggestion
- **Scaling Concurrency**: You can run up to 80 concurrent jobs on our monthly Performance Plan or even more on a [custom plan](https://circleci.com/pricing/). Travis CI has capped concurrencies of 1, 2, 5, and 10 on each of their plans.
```
I would refer to our "annual" plan as "custom" as that is the name of the plan (that happens to have a customizable annual billing cycle))